src/js/components/mon-compte/export-credentials-component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { InputLabel, Select, MenuItem } from '@material-ui/core';
+
+class ExportCredentialsField extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = { value: 'python' };
+		this.credentials = props.credentials;
+	}
+
+	handleChange = (e) => this.setState({ value: e.target.value });
+
+	render() {
+		const pythonText = `ceci est l'export en Python : ${this.credentials.AWS_ACCESS_KEY_ID}`;
```

Comment:
See previous comment, I think it's better to access credentials using `this.props.credentials`

Reply:
Done